Classes/FusionObjects/ForPrototypeImplementation.php

```diff
@@ -0,0 +1,18 @@
+<?php
+declare(strict_types=1);
+
+namespace PackageFactory\AtomicFusion\PropTypes\FusionObjects;
+
+use Neos\Fusion\FusionObjects\AbstractFusionObject;
+use PackageFactory\AtomicFusion\PropTypes\Validators\DataStructureValidator;
+
+class ForPrototypeImplementation extends AbstractFusionObject
+{
+    use AbstractValidatorTrait;
+
+    public function createValidator(): DataStructureValidator
+    {
+        $prototypeName = $this->fusionValue('prototypeName');
```

Comment:
i would argue that "prototype" is the wrong name, its too unspecific propTypes can only be used with components ... so how about ForComponent and componentName ?

Reply:
Used it as it is the same as in Sitegeist.Monocle:PreviewPrototype ... bettter suggestions are welcome. Also it would be great to specify a prototype not as string. Did not find a good way to achieve this.

Reply:
i now some dark magic so you can do 

type = Foo.Bar
 (without strings ^^)

but thats not really tge fusion way, i think people got used to it ...

Reply:
Und dann `runtime->evaluate->("type/__meta/propTypes");` ... hmmm really dark

Reply:
hmm one can access the object name via `$this->properties["type"]['__objectType']` ;)

(tested)

Reply:
@mhsdesign Would still like to go with "string:prototypeName" first. If we find ab better way we can implement the same solution in monocle and here.